src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -524,12 +541,18 @@
         flex: 0 0 auto;
       }
     }
+
+    .datagrid-column:last-child {
+      .datagrid-column-separator {
+        display: none;
+      }
+    }
   }
 
   .datagrid-row-flex {
     flex: 1 1 auto;
     display: flex;
-    flex-flow: column nowrap;
+    flex-flow: row nowrap;
```

Comment:
I need the elements to flex as rows on top of each other to make the row-detail element work here.